README.md

```diff
@@ -30,6 +30,8 @@ file. For an installation from scratch, the commands would be:
 
     git clone -b 13.x http://github.com/claroline/Claroline
     cd Claroline
+    php bin/check
```

Comment:
`bin/check` fails if we run it before `bin/configure`, thus blocking the build, because of a missing parameters.yml:

> Pre-install check failed: The configuration file config/parameters.yml is missing (execute "php scripts/configure.php" to build it interactively)

Shall we run it after bin/configure instead? (it seems to work for me)
```
php bin/configure
php bin/check
```
Or shall it not fail when there's no parameters.yml file yet?